installers/MacInstaller/make_package.rb

```diff
@@ -28,43 +27,49 @@
   'wheel',
   'wheel*.*-info'
 ].freeze
-# Brew Python packages to be copied to bundle
-BUNDLED_PY_MODULES_MANTIDPLOT = [
-  'PyQt4/__init__.py',
-  'PyQt4/Qt.so',
-  'PyQt4/QtCore.so',
-  'PyQt4/QtGui.so',
-  'PyQt4/QtOpenGL.so',
-  'PyQt4/QtSql.so',
-  'PyQt4/QtSvg.so',
-  'PyQt4/QtXml.so',
-  'PyQt4/sip.so',
-  'PyQt4/uic'
-].freeze
 BUNDLED_PY_MODULES_WORKBENCH = [
   'PyQt5/__init__.py',
-  'PyQt5/Qt.so',
-  'PyQt5/QtCore.so',
-  'PyQt5/QtGui.so',
-  'PyQt5/QtOpenGL.so',
-  'PyQt5/QtPrintSupport.so',
-  'PyQt5/QtSql.so',
-  'PyQt5/QtSvg.so',
-  'PyQt5/QtTest.so',
-  'PyQt5/QtWidgets.so',
-  'PyQt5/QtXml.so',
-  'PyQt5/sip.so',
+  'PyQt5/Qt.*so',
+  'PyQt5/QtCore.*so',
+  'PyQt5/QtGui.*so',
+  'PyQt5/QtOpenGL.*so',
+  'PyQt5/QtPrintSupport.*so',
+  'PyQt5/QtSql.*so',
+  'PyQt5/QtSvg.*so',
+  'PyQt5/QtTest.*so',
+  'PyQt5/QtWidgets.*so',
+  'PyQt5/QtXml.*so',
+  'PyQt5/sip.*so',
   'PyQt5/uic',
 ].freeze
-REQUIREMENTS_FILE = Pathname.new(__dir__) + 'requirements.txt'
-REQUIREMENTS_WORKBENCH_FILE = Pathname.new(__dir__) + 'requirements-workbench.txt'
+# Python requirements. Versions hard pinned to those
+# used by the host at install time
+REQUIREMENTS=[
+  "h5py",
```

Comment:
It will yep. I've actually removed one of the locations that specifies the versions now. I deleted the requirements files that were used in packaging in this PR and have altered the script to tell `pip` to use those versions that are installed in the developer environment so that atleast now they won't drift.
I could move `requirements.txt` from https://github.com/mantidproject/homebrew-mantid but it feels like that is better versioned with that repo. As you say, this all goes away with conda...